src/test/kotlin/com/zepben/evolve/cim/iec61970/base/core/CurveTest.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.cim.iec61970.base.core
+
+import com.zepben.evolve.services.network.NetworkService
+import com.zepben.evolve.services.network.testdata.fillFields
+import com.zepben.testutils.junit.SystemLogExtension
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.hamcrest.Matchers.not
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+
+internal class CurveTest {
+
+    @JvmField
+    @RegisterExtension
+    var systemErr: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
+
+    @Test
+    internal fun constructorCoverage() {
+        assertThat(object : Curve() {}.mRID, not(equalTo("")))
+        assertThat(object : Curve("id") {}.mRID, equalTo("id"))
+    }
+
+    @Test
+    internal fun accessorCoverage() {
```

Comment:
todo: No need for the accessor coverage test for this class as private collections are tested elsewhere and there are no other accessors.

Reply:
removed